selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/utils/BrowserLogsUtil.java

```diff
@@ -25,27 +27,115 @@
  * Read and store browser logs to the test logs. Log level and type are defined in {@link
  * org.eclipse.che.selenium.core.SeleniumWebDriver#doCreateDriver(URL)}
  */
-@Singleton
 public class BrowserLogsUtil {
-  private final SeleniumWebDriver seleniumWebDriver;
-  private final Logger LOG = LoggerFactory.getLogger(BrowserLogsUtil.class);
-
-  @Inject
-  public BrowserLogsUtil(SeleniumWebDriver seleniumWebDriver) {
-    this.seleniumWebDriver = seleniumWebDriver;
-  }
+  private static final Logger LOG = LoggerFactory.getLogger(BrowserLogsUtil.class);
 
   /**
    * read logs from browser console
    *
    * @return log messages from browser console
    */
-  public List<LogEntry> getLogs() {
+  public static List<LogEntry> getConsoleLogs(WebDriver seleniumWebDriver) {
     return seleniumWebDriver.manage().logs().get(BROWSER).getAll();
   }
 
+  /**
+   * get all logs from the active webdriver session
+   *
+   * @return all types of performance logs
+   */
+  public static List<LogEntry> getPerformanceLogs(WebDriver seleniumWebDriver) {
+    return seleniumWebDriver.manage().logs().get(PERFORMANCE).getAll();
+  }
+
   /** store browser logs to the test logs */
-  public void storeLogs() {
-    getLogs().forEach(logEntry -> LOG.info("{} {}", logEntry.getLevel(), logEntry.getMessage()));
+  public static void storeLogsToConsoleOutput(WebDriver seleniumWebDriver) {
+    getConsoleLogs(seleniumWebDriver)
+        .forEach(logEntry -> LOG.info("{} {}", logEntry.getLevel(), logEntry.getMessage()));
+  }
+
+  /**
+   * combine the Network and Browser logs
+   *
+   * @return logs from browser console and requests/responses on CHE api
+   */
+  public static String getCombinedLogs(WebDriver seleniumWebDriver) {
+    StringBuilder combinedLogs =
+        new StringBuilder("Browser console logs:\n").append("---------------------\n");
+    getConsoleLogs(seleniumWebDriver)
+        .forEach(
+            logEntry ->
+                combinedLogs
+                    .append(String.format("%s  %s \n", logEntry.getLevel(), logEntry.getMessage()))
+                    .append("\n"));
+    return combinedLogs.append(getNetworkDataSentOnCheApi(seleniumWebDriver)).toString();
+  }
+
+  /** filter data and get requests/responses that has been sent on CHE /api/ URL */
+  public static String getNetworkDataSentOnCheApi(WebDriver seleniumWebDriver) {
```

Comment:
I don't see filtering of Che API specific logs.
`getNetworkDataSentOnCheApi` > `readNetworkLogs`